docs/topics/proc_deploying-the-configmap-booster-using-the-oc-cli-client.adoc

```diff
@@ -6,11 +6,12 @@
 
 
 [id='deploying-the-configmap-booster-using-the-oc-cli-client_{context}']
-= Deploying the {name-mission-configmap} booster using the `oc` CLI client
+[id='deploying-the-configmap-example-using-the-oc-client_{context}']
+= Deploying the {name-mission-configmap} example using the `oc` CLI client
```

Comment:
Deploying the {name-mission-configmap} example using --> Deploying the {name-mission-configmap} example application using

Reply:
Updated similar modules as well.